beacon-chain/casper/validator.go

```diff
@@ -13,10 +13,10 @@ const bitsInByte = 8
 
 // ActiveValidatorIndices filters out active validators based on start and end dynasty
 // and returns their indices in a list.
-func ActiveValidatorIndices(validators []*pb.ValidatorRecord, dynasty uint64) []uint32 {
+func ActiveValidatorIndices(validators []*pb.ValidatorRecord) []uint32 {
 	var indices = make([]uint32, 0, len(validators))
 	for i := 0; i < len(validators); i++ {
-		if validators[i].StartDynasty <= dynasty && dynasty < validators[i].EndDynasty {
+		if validators[i].Status == uint64(params.Active) {
```

Comment:
can/should the validator status enums be cast as `uint64` to begin with?

Reply:
+1